index.js

```diff
@@ -0,0 +1,90 @@
+// Popup Nav
+
+function openPopUp() {
+  document.querySelector('.popup-nav').classList.add('popup-nav-visible');
+}
+
+function closePopUp() {
+  document.querySelector('.popup-nav').classList.replace('popup-nav-visible', 'popup-nav');
+}
+
+document.querySelector('.menu').addEventListener('click', openPopUp);
+document.querySelector('.cancel-button').addEventListener('click', closePopUp);
+document.querySelector('.popup-menu .menu-item').addEventListener('click', closePopUp);
+document.querySelector('.popup-menu .menu-item2').addEventListener('click', closePopUp);
+document.querySelector('.popup-menu .menu-item3').addEventListener('click', closePopUp);
+document.querySelector('.popup-nav').addEventListener('click', closePopUp);
+
+// Featured Speakers
+
+const speakersData = [
+  {
+    id: 1,
+    name: 'John Smith',
+    FeaturedImage: 'images/speaker_1.svg',
+    Description: 'Experienced educator with a Bachelor\'s degree in Education',
+    fullDescription: 'Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006',
+  },
+  {
+    id: 2,
+    name: 'Jane Doe',
+    FeaturedImage: 'images/speaker_2.svg',
+    Description: 'Highly qualified teacher with a Master\'s degree in Educational Leadership',
+    fullDescription: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Odio a nostrum maiores deserunt expedita quae cum eligendi, cupiditate nesciunt voluptates voluptatum',
+  },
+  {
+    id: 3,
+    name: 'Mary Brown',
+    FeaturedImage: 'images/speaker_3.svg',
+    Description: ' Accomplished professor with a Bachelor\'s degree in Special Education',
+    fullDescription: 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Hic voluptates veniam quos minima. Quaerat perspiciatis, animi magni minus velit provident neque aperiam laborum.',
```

Comment:
- [ ] According to the project requirements the implementation of `more` button is optional however please make sure to in-list all speakers in the mobile resolution

<img width="367" alt="Screenshot 2023-03-18 at 1 18 48 PM" src="https://user-images.githubusercontent.com/44798044/226094120-9c96740e-12c9-4f75-a384-b04185ad2af8.png">